docs/spelling_wordlist.txt

```diff
@@ -1,2 +1,4 @@
 Ansible
 ansible
+stderr
```

Comment:
Is this only used in the change note? I'd suggest rephrasing/expanding instead because the change notes are targeting the end-user/human audience and the generic style guidance for this type of docs is to use full words + avoid abbreviations.